Deprecated `inject.bundle.js`, use `redux-devtools-extension.js` instead

jhen0409 opened the pull request:
To avoid problems like https://github.com/jhen0409/react-chrome-extension-boilerplate/issues/50, we can provide a better filename for inject bundle.

zalmoxisus commented:
Thanks @jhen0409 for addressing this!

In perspective we'd like to remove this extra file, so it would be great to add a warning in `inject.bundle.js` like
`Using 'chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/inject.bundle.js' is deprecated. Use 'chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/redux-devtools-extension.js' instead.`

jhen0409 commented:
@zalmoxisus, I just updated for that, I added [deprecatedWarn.js](https://github.com/jhen0409/redux-devtools-extension/blob/591b9cc0a2ba72467b671413196cf427b515ab70/src/browser/extension/inject/deprecatedWarn.js) for inject bundle, so it will has two entries (inject, redux-devtools-extension) for webpack config.

zalmoxisus commented:
@jhen0409, good work, as always! I'll publish it to Chrome Store tomorrow.
